packages/agent/src/agent/http/index.ts

```diff
@@ -1386,6 +1406,24 @@ export class HttpAgent implements Agent {
 
     return p;
   }
+
+  /**
+   * Returns the time difference in milliseconds between the client's clock and the IC network clock,
+   * after the clock has been synced using the {@link HttpAgent.syncTime} method
+   * or during agent creation if {@link HttpAgentOptions.shouldSyncTime} was set to `true`.
```

Comment:
```suggestion
   * after the clock has been synced.
```

I wouldn't go into details when time is synced since the details might soon get outdated.